english/dragons/fire-everywhere.md

```diff
@@ -42,7 +42,11 @@ Drogon to fly...
 You do not much energy left in you...You are extremely tired...You don't even realize when you 
 fell asleep...
 
-When you wake up its noon and you come to know that it was all a dream !
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.